sdk/rust/oak_tests/src/lib.rs

```diff
@@ -74,6 +76,21 @@ pub fn run_single_module(
     module_config_name: &str,
     entrypoint_name: &str,
 ) -> Result<(oak_runtime::RuntimeProxy, oak_abi::Handle), oak::OakStatus> {
+    let (app_config, runtime_config, grpc_config) =
+        runtime_configs(module_config_name, entrypoint_name);
+    oak_runtime::configure_and_run(app_config, runtime_config, grpc_config)
+}
+
+/// Build the set of configurations needed to launch a test Runtime instance that
+/// runs a single-Node application with the given module name and entrypoint.
+pub fn runtime_configs(
+    module_config_name: &str,
+    entrypoint_name: &str,
+) -> (
+    ApplicationConfiguration,
+    oak_runtime::RuntimeConfiguration,
+    oak_runtime::GrpcConfiguration,
```

Comment:
I think we should consider a proper struct for this, not just for tests, but even for the actual runtime / loader code.

One possibility is to just extend `RuntimeConfiguration` to have fields for `ApplicationConfiguration` and `GrpcConfiguration`.

Reply:
Yeah, I was being lazy with that &ndash; done.